.github/workflows/pull_requests.yaml

```diff
@@ -104,27 +108,22 @@ jobs:
           WCS_DUMMY_CI_PW: ${{ secrets.WCS_DUMMY_CI_PW }}
           WCS_DUMMY_CI_PW_2: ${{ secrets.WCS_DUMMY_CI_PW_2 }}
         run: ./test/run.sh --acceptance-only
-  Codecov:
```

Comment:
Why is codecov step removed? I remember that Dirk made a [PR](https://github.com/weaviate/weaviate/pull/3569) where he set this step to not to fail the CI if it is red.

Reply:
It was covering the page in useless messages that made the diff impossible to read

Reply:
We all get those annotations, would be good to first discuss this change and if we would agree to completely remove this then we would do this in a separate PR.